pymc/distributions/continuous.py

```diff
@@ -777,11 +777,13 @@ def logp(
             norm = 0.0
 
         logp = _logprob(normal, (value,), None, None, None, mu, sigma) - norm
+        logp = at.switch(
+            at.or_(at.le(value, lower), at.ge(value, upper)),
```

Comment:
I think the bounds are part of the domain:

```suggestion
            at.or_(at.lt(value, lower), at.gt(value, upper)),
```

Reply:
Good catch!